homework-submission/react-hyf/react-hyf-client/src/App.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PreviousHomeworks from './components/previousHomeworks/previousHomeworks';
+import HomeworkWeek3 from './components/Users/Users';
+
+class App extends React.Component {
+  state = {
+    data: [],
+    isLoading: true,
+    error: '',
+  };
+
+  componentDidMount() {
+    fetch('https://uinames.com/api/?amount=10')
```

Comment:
I like to put API calls in a separate service/API file and import them here. You could have a function in that file like: 

```
const url = https://uinames.com/api/?amount=10

export function getNames(){
return fetch(url).then(data => data.json())
}
```